tools/check_version_bump.py

```
@@ -107,14 +125,40 @@ def check(base_ref: str | None = None) -> tuple[int, list[str]]:
+    # committed edit whose bump is still uncommitted passes here and fails CI,
+    # and an untracked scratch file under skills/ fails here and is invisible
+    # to CI. The local answer is about the tree you are about to land; CI's is
+    # about the commits you did.
+    for args in (("diff", "--name-only", "--no-renames", "HEAD"),
+                 ("ls-files", "--others", "--exclude-standard")):
+        code, out, err = _git(*args)
+        if code != 0:
+            return UNDETERMINED, [
+                "version-bump (ADR-003): could not read the working tree "
+                f"({' '.join(args)} failed{': ' + err if err else ''}) — "
+                "refusing to answer from committed history alone"
```

Ruff (0.16.1)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)